packages/angular/src/tracing.ts

```diff
@@ -224,7 +225,7 @@ export function TraceMethodDecorator(): MethodDecorator {
         activeTransaction.startChild({
           description: `<${target.constructor.name}>`,
           endTimestamp: now,
-          op: `angular.${String(propertyKey)}`,
+          op: `${ANGULAR_OP}.${String(propertyKey)}`,
```

Comment:
Same thing as React comments, especially since we recommend low cardinality ops, we should probably only use the same exported `const` ops and consume them here.

Reply:
I agree, but not sure how to best start considering: 1) we have 0 tests for angular (lol) and 2) this matches existing behaviour.

The `propertyKey` should be a set of strings though, not uuids or unique names, so I think we still fit the low cardinality aspect.